commons/src/main/java/com/powsybl/commons/json/JsonUtil.java

```diff
@@ -479,6 +479,19 @@ public static void assertLessThanReferenceVersion(String contextName, String ele
         }
     }
 
+    public static int compareVersions(String v1, String v2) {
+        String[] parts1 = v1.split("\\.");
```

Comment:
Since previously the comparison was made one Strings, which can accept any value, I think we should ensure that the method will not throw an exception when the version are not real version numbers.

So I think we could add the following securities:
- add a try/catch around the block checking the major versions
  - if a `NumberFormatException` is caught, return `v1.compareTo(v2)` (similar to the existing)
- add a try/catch around the block checking the minor versions
  - if a `NumberFormatException` is caught, return `minor1.compareTo(minor2)`

<br/>

Can you also add the following checks at the beginning of the method?
```java
Objects.requireNonNull(v1);
Objects.requireNonNull(v2);
```